tools/distrib/python/make_grpcio_tools.py

```diff
@@ -228,7 +231,19 @@ def _copy_source_tree(source, target):
             shutil.copyfile(source_file, target_file)
 
 
+def _delete_source_tree(target):
+    """Deletes the copied target directory."""
+    target = GRPCIO_TOOLS_ROOT_PREFIX + target
+    target_abs = os.path.join(*target.split("/"))
+    print("Deleting copied folder %s" % (target_abs))
+    shutil.rmtree(target_abs, ignore_errors=True)
+
 def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--cleanup-third-party", default=False, action="store_true", help="Delete the temporary third_party folder"
```

Comment:
nit: naming:  throughout the grpc repo ,we use underscores in flag names AFAICT. So `--cleanup_third_party` would be more consistent?